src/main/java/com/denizenscript/depenizen/bukkit/objects/vivecraft/ViveCraftPlayerTag.java

```diff
@@ -176,5 +172,34 @@ public static void register() {
             }
             return new LocationTag(location);
         });
+
+        // <--[tag]
+        // @attribute <ViveCraftPlayerTag.rotation[head/left/right]>
+        // @returns QuaternionTag
+        // @description
+        // Returns a QuaternionTag of the rotation of the input player part.
+        // -->
+        tagProcessor.registerTag(QuaternionTag.class, ElementTag.class, "rotation", (attribute, object, input) -> {
+            float[] rotation;
+            switch (input.toString()) {
+                case "head":
+                    rotation = (float[]) object.getPlayer().getMetadata("head.rot").get(0).value();
+                    break;
+                case "left":
+                    rotation = (float[]) object.getPlayer().getMetadata("lefthand.rot").get(0).value();
+                    break;
+                case "right":
+                    rotation = (float[]) object.getPlayer().getMetadata("righthand.rot").get(0).value();
+                    break;
+                default:
+                    attribute.echoError("Attribute must be 'head', 'left' or 'right'");
```

Comment:
LGTM, just one nitpick - don't think `Attribute` is right here? can probably say `input`, `input part`, or format it like `Invalid player part specified, must be 'head', 'left', or 'right'.` or something like that